async/retry.ts

```diff
@@ -64,35 +93,36 @@ export async function retry<T>(
     options.minTimeout <= options.maxTimeout,
     "minTimeout is greater than maxTimeout",
   );
+  assert(options.jitter <= 1, "jitter is greater than 1");
```

Comment:
Maybe we should also assert `options.jitter >= 0`?

Reply:
I thought about that, but didn't seem necessary to me. It would simply increase the delay beyond the minimum then.